nrf_rpc/doc/architecture.rst

```diff
@@ -67,7 +67,7 @@ The following image presents sample command and response flows.
 
    nRF RPC simple command flow
 
-A thread waiting for a response can be reused to receive a new incoming command from the remote thread that the local thread is waiting for, e.g. when a callback is called synchronously.
+A local thread waiting for a response can be reused to receive a new incoming command from the remote thread, for example, when a callback is called synchronously.
```

Comment:
Removed part of the sentence specifies which remote thread can reuse the local thread. In modified version it is not clear which remote thread is allowed to reuse local, so the someone, who reads it, may think that any remote thread can reuse the local thread.

Reply:
Rephrased as - A thread waiting for a response can be reused to receive a new incoming command from the remote thread for which the local thread is waiting, for example, when a callback is called synchronously.

Reply:
That looks good.